research/flamby/utils.py

```diff
@@ -109,8 +109,8 @@ def evaluate_fed_isic_model(
 ) -> float:
     meter = evaluate_model_on_dataset(model, dataset, metrics, device, is_apfl)
     computed_metrics = meter.compute()
-    assert "test_meter_FedIsic2019_balanced_accuracy" in computed_metrics
-    balanced_accuracy = computed_metrics["test_meter_FedIsic2019_balanced_accuracy"]
+    assert "test_meter - prediction - FedIsic2019_balanced_accuracy" in computed_metrics
+    balanced_accuracy = computed_metrics["test_meter - prediction - FedIsic2019_balanced_accuracy"]
```

Comment:
For some reason (probably due to code refactor) I needed to change these evaluation files to make it work.

Reply:
Yeah, we made some changes to the way this stuff is named in the metrics classes. I'm glad you were able to fix it in a simple way.